content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
 * Work in teams of 2+ mentors per 1 each student
 * Dedicate a consistent and significant amount of time, especially during the coding phase (_~5 hours_ per week in a team of two mentors)
 
-Mentorship does **NOT** require strong expertise in Jenkins plugin development.
+Mentoring does **NOT** require strong expertise in Jenkins plugin development.
 The main objective is to guide students and to get them involved into the Jenkins community.
-GSoC org admins will help to find advisors if a special expertise is needed.
+GSoC org admins will help to find advisers if a special expertise is needed.
+
+== Expectations from mentors per phase
+
+We have divided what mentors should expect in the different phases.
+Overall for mentors the time commitment is:
+
+* Organization application phase: spend the time you want.
+* Student exploration and application phase: 2 to 5 hours per week for 5.5 weeks.
+* Slot request and final selection phase: about 5 hours total over 3 weeks, but because this phase is time critical and short, we insist that you make an extra effort to attend meetings and participate actively in this process (see below for details).
+* Community bonding: 5 to 8 hours per week, this phase is critical for the success of the project, make sure you get on the ball right away.
+* Coding and evaluation periods: 5 to 8 hours per week until the end of the program. The number of hours tend to vary, more at the start, less at the end, not everyone is the same.
+
+=== Call for mentors and organization application phase
```

Comment:
Is this meant to be the same as `Student exploration and application phase` above? If so, could we make them consistent?

Reply:
Never mind, it makes sense after reading further